src/object.c

```diff
@@ -692,28 +692,35 @@ char *strEncoding(int encoding) {
 
 /* =========================== Memory introspection ========================== */
 
+static size_t sizeOfStringObject(robj *o) {
+    size_t strSize = 0;
+    if (o->encoding == OBJ_ENCODING_INT) {
+        strSize = sizeof(long long);
```

Comment:
Normally, o->ptr save an address which points to other address, in which case we should take that into account.
But here o->ptr just save a long long value instead of other address.
So sizeof(long long) is redundant.
Fix that. Thanks :)